engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/compile/NodeDataValidatorSpec.scala

```diff
@@ -132,8 +134,29 @@ class NodeDataValidatorSpec extends FunSuite with Matchers with Inside {
     }
   }
 
-  ignore("should validate variable builder definition") {
-    //TODO
+  test("should return expression type info for variable definition") {
+    inside(
+      validate(Variable("var1", "var1", "42L", None), ValidationContext(Map.empty))
+    ) {
+      case ValidationPerformed(Nil, _, Some(expressionType)) => expressionType.display shouldBe "Long"
+    }
+  }
+  test("should validate variable builder definition") {
```

Comment:
I think we can also include test case for checking if variable name is ok

Reply:
👍